lxd/db/cluster/schema.go

```diff
@@ -532,6 +532,7 @@ CREATE TABLE storage_volumes_snapshots (
     storage_volume_id INTEGER NOT NULL,
     name TEXT NOT NULL,
     description TEXT,
+    expiry_date DATETIME,
```

Comment:
@monstermunchkin is the `expiry_date` column name a historical legacy, or is there the option of choosing a different name that better indicates that the resolution of the expiry is not just the date, but in fact the time as well. Perhaps `expire_at` or `expiry_datetime`?

Reply:
Historical legacy. Instances have the same field, and there are other dates as well, e.g. `creation_date`.

Reply:
@monstermunchkin ok, so guess we'll have to stick with it then, better a consistent confusion a reader only has to learn once than a series of inconsistent naming :)

At least the function arguments can avoid the Date suffix though.